README.md

```diff
@@ -245,3 +245,22 @@ Branch names are enforced before you even commit to ensure everyone adopts the s
 For end to end testing, we use the notorious [cypress](https://github.com/cypress-io/cypress).
 
 We also have a pre-built CI toolkit for you that will build and run the tests.
+
+# Overview
+
+Stator is a full-stack pre-configured template for your projects.
+The technologies used are mentioned below-
+* Database
+  * Postgres
+  * Mongo
+  * Seeding data
+* Backend
+    * TypeORM
+    * Nest
+    * Fastify
+    * NestJs CRUD
+* Frontend
+    * React
+    * Redux Toolkit
+    * React Router
+    * axios    
```

Comment:
I think it would be great to add links to all the projects.

Reply:
Okay, working on the suggestions.